CMakeLists.txt

```diff
@@ -221,8 +219,10 @@ set(libtorrent_aux_include_files
 	invariant_check
 	instantiate_connection
 	io
+	io_bytes
```

Comment:
@arvidn naming might not be great, any suggestion?

Reply:
this seems fine